sale_cancel_remaining/wizards/cancel_remaining_wizard.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2018 Okia SPRL
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, models
+from odoo.exceptions import UserError
+
+
+class CancelRemainingWizard(models.TransientModel):
+    _name = "cancel.remaining.wizard"
```

Comment:
Maybe safer to bind the wizard to the model it applies to
```suggestion
    _name = "sale.order.line.cancel"
```